angular-training/src/app/components/component-three/component-three.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { SharingService } from '../../services/Sharing/sharing-service.service';
+import { Router, RouterModule } from '@angular/router';
+import { MatButtonModule } from '@angular/material/button';
+import { FormsModule } from '@angular/forms';
+
+@Component({
+  selector: 'app-component-three',
+  standalone: true,
+  imports: [RouterModule, MatButtonModule, FormsModule],
+  providers: [RouterModule],
+  templateUrl: './component-three.component.html',
+  styleUrl: './component-three.component.css'
+})
+export class ComponentThreeComponent {
+  data:any = "example";
```

Comment:
Avoid any type as much as possible. If needed, you can use generics for data passing. In this case you can use type inference because typescript can identify its a string when you are initializing it.